Day-4/src/components/Cards.js

```diff
@@ -0,0 +1,27 @@
+import React from "react";
+import "./styles.css";
+
+const Cards = ({ data }) => {
+  console.log(data);
+  return (
+    <div className="card-list">
+      {data &&
+        data.map((item) => {
+          return (
+            <div className="member-card" key={item.id}>
+              <div className="member-details">
+                <div>{`Team Member #${item.id}`}</div>
+                <h1 className="member-name">{item.name}</h1>
+                <h3 className="member-role">{item.designation}</h3>
```

Comment:
Unused classes can be removed. You need not give every element class, some can be handled within CSS also.
Like 
.member-details > h1{
    // some style here
}

Reply:
done!